index.bs

```diff
@@ -324,18 +324,15 @@ type: dfn
         [=potentially trustworthy origin=], then return
         "`Prohibits Mixed Security Contexts`".
 
-    2.  If |settings| has a <a>responsible document</a> |document|, then:
+    2.  If |settings|' [=environment settings object/global object=] has an
+        [=Window/associated Document=] |document|, then:
```

Comment:
It'd be slightly more correct to ask if settings's global object is a Window, and only then look at its associated Document. But what you have here seems reasonable too.

Reply:
Done, thanks for reviewing.